libraries/BufferedBleSpp/nanoFramework/BufferedBleSpp.cs

```diff
@@ -0,0 +1,275 @@
+﻿//
+// Copyright (c) .NET Foundation and Contributors
+// See LICENSE file in the project root for full license information.
+//
+
+using System;
+using nanoFramework.Device.Bluetooth.GenericAttributeProfile;
+using System.Diagnostics;
+using System.Text;
+using nanoFramework.Device.Bluetooth.Spp;
+using System.IO;
+using System.Collections;
+using nanoFramework.Device.Bluetooth;
+
+namespace nanoFramework.Device.BufferedBleSpp
+{
+    /// <summary>
+    /// Implementation of a SPP profile which can handle large buffer sizes.
+    /// </summary>
+    public class BufferedBleSpp
+    {
+        private const int BUFFER_SIZE = 250;
+
+        private string deviceName;
+
+        // UUID for UART service
+        private Guid ServiceUUID = new Guid("12345678-1234-5678-1234-56789abcdef0");

```

Comment:
I guess all those can be readonly

Reply:
And question I'm raising here: would it make more sense to have a shared project where you can place all those common elements? So you can share them between the nano and Xamarin solutions